indico/modules/events/controllers/creation.py

```diff
@@ -46,16 +46,24 @@ class RHCreateEvent(RHProtected):
     def _process_args(self):
         self.event_type = EventType[request.view_args['event_type']]
         self.root_category = Category.get_root()
-        self.single_category = not self.root_category.children
+        self.single_category = not self.root_category.has_children
```

Comment:
It's only used in one place, so I'd move the `self.root_category.has_children` check down there